galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager.ivt/src/main/java/dev/galasa/cicsts/manager/ivt/CICSTSManagerIVT.java

```diff
@@ -0,0 +1,199 @@
+	   logger.info("Testing that the CICS TS Manager finds the DSE CICS job for the CICS Region");
+	   logger.info("There should be an active job as the CICS Region is running");
+	   boolean exceptionThrown = false;
+	   IZosBatchJob regionJob = null;
+	   try {
+		   regionJob = cics.getRegionJob();
+	   } catch (CicstsManagerException e) {
+		   exceptionThrown = true;
+	   }
+	   assertThat(exceptionThrown).isFalse();
+	   assertThat(regionJob).isNotNull();	   
+   }
+   
+   /**
+    * Tests that the CICS Terminal in the CICS TS Manager retrieves the correct CICS Region
+    * @throws FieldNotFoundException 
+    * @throws NetworkException 
+    * @throws TerminalInterruptedException 
+    * @throws KeyboardLockedException 
+    * @throws TimeoutException 
+    */
+   @Test
+   public void testGetCicsRegion() throws TimeoutException, KeyboardLockedException, TerminalInterruptedException, NetworkException, FieldNotFoundException {
+	   logger.info("Testing that the CICS Terminal gets the correct CICS Region");
+	   String testCicsRegion = terminal.getCicsRegion().toString().replace("CICS Region[", "").replace("]", "");
+	   logger.info("CICS Region retrieved by the ICicsTerminal: " + testCicsRegion);
+	   terminal.type("CEMT INQUIRE").enter().waitForKeyboard();
+	   assertThat(terminal.retrieveScreen().contains(testCicsRegion)).isTrue();
+   }
+   
+   /**
+    * Tests that the CICS Terminal in the CICS TS Manager correctly connects to the CICS Region
+    * @throws CicstsManagerException
+    * @throws TerminalInterruptedException 
+    * @throws NetworkException 
+    * @throws KeyboardLockedException 
+    * @throws TimeoutException 
+    */
+   @Test
+   public void testConnectToCicsRegion() throws CicstsManagerException, TerminalInterruptedException, KeyboardLockedException, NetworkException, TimeoutException {
+	   logger.info("Testing that the CICS Terminal connects to the CICS Region");
+	   logger.info("First, need to disconnect from the CICS Region");
+	   terminal.disconnect();
+	   assertThat(terminal.isConnected()).isFalse();
+	   terminal.connectToCicsRegion();
+	   assertThat(terminal.isConnected()).isTrue();
+	   assertThat(terminal.isClearScreen()).isTrue();
+   }
+   
+   /**
+    * Tests that the CICS Terminal in the CICS TS Manager resets and clears correctly
+    * @throws FieldNotFoundException 
+    * @throws NetworkException 
+    * @throws TerminalInterruptedException 
+    * @throws KeyboardLockedException 
+    * @throws TimeoutException 
+    */
+   @Test
+   public void testResetAndClear() throws TimeoutException, KeyboardLockedException, TerminalInterruptedException, NetworkException, FieldNotFoundException {
+	   logger.info("Testing that the CICS Terminal resets and clears screen correctly");
+	   boolean exceptionThrown = false;
+	   try {
+		   terminal.resetAndClear();
+	   } catch (CicstsManagerException e) {
+		   exceptionThrown = true;
+	   }
+	   assertThat(exceptionThrown).isFalse();
```

Comment:
same comment as earlier - we could declare the exception in the method signature and use some logging to say what we are doing

Reply:
Same as my previous response - I have removed the try catch block and added logging to make it clearer what the test is attempting to do. If an exception is thrown, it is obvious that whatever we were trying to do failed.